server/server.js

```diff
@@ -38,11 +38,6 @@ app.use((err, req, res, next) => {
 app.use(compression());
 
 const server = app.listen(PORT, function () {
-  console.log(config.ENV);
-  if (config.ENV === config.TEST) {
```

Comment:
To clarify: I intended this to be here for testing with `yarn start_test` which uses the emulated firestore DB rather than our live DB.